doc/source/getting_started/intro_tutorials/01_table_oriented.rst

```diff
@@ -176,7 +176,7 @@ these are by default not taken into account by the :func:`~DataFrame.describe` m
 
 Many pandas operations return a ``DataFrame`` or a ``Series``. The
 :func:`~DataFrame.describe` method is an example of a pandas operation returning a
-pandas ``Series``.
+pandas ``Series`` (one numerical column) or a pandas ``DataFrame`` (many numerical columns).
```

Comment:
Do we need the column reference? Meaning is this done elsewhere in the getting started docs?